### ISSUE TITLE
Accessibility regression: cannot click buttons using Home Assistant companion for Android with screen reader

### ISSUE THREAD

**Issue by Nardol** [home-assistant/frontend#10617]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Using Android Home Assistant app with TalkBack, when going to configuration then server control, I cannot click any button, for example restart or check config.
Moreover, I have every button twice and none can be used.
Using Chrome on a desktop I still have buttons twice but the second one is recognized as a button and can be clicked.

### Describe the behavior you expected

I should be able to double-tap the button to activate it.

### Steps to reproduce the issue

1. Open Android App with TalkBack enabled by placing your finger on Home Assistant then double-tapping
2. Activate side bare by placing the finger on then double-tapping the button to show it
3. Place a finger then double-tap configuration
4. Place the finger to server control then double-tap
5. Put your finger on a button, then double-tap

### What version of Home Assistant Core has the issue?

2021.11

### What was the last working version of Home Assistant Core?

2021.10

### In which browser are you experiencing the issue with?

Home Assistant android app

### Which operating system are you using to run this browser?

Android 12

### Additional information

It is a really annoying accessibility regression which make unusable some elements of Home Assistant interface, it looks like others buttons are no more recognized by TalkBack

**Comment by spacegaier** [comment]
I never tried the app with a screen reader, but we had made a big change in 2021.11 in regards to the icon buttons specifically to improve the accessibility by ensuring that all (or realistically most) now have an aria-label as confirmed by Chrome Lighthouse tests.

Not sure though if that has any link to this issue, since you mention buttons such as "Check config" or "Restart" which are actual text buttons, whereas the change I mentioned is for icon buttons (so no text, purely an icon on the button).

Reg "Using Chrome on a desktop I still have buttons twice": You mean in Chrome they are rendered twice or the screen reader is listing/saying them twice (if you can verify that distinction at all)?

**Comment by Nardol** [comment]
> I never tried the app with a screen reader, but we had made a big change in 2021.11 in regards to the icon buttons specifically to improve the accessibility by ensuring that all (or realistically most) now have an aria-label as confirmed by Chrome Lighthouse tests.
> 

Nice to know this excellent news :slightly_smiling_face: I had not read it in any changelog I read until now.
I am really happy to know accessibility has been taken in account more than I thought.

> Not sure though if that has any link to this issue, since you mention buttons such as "Check config" or "Restart" which are actual text buttons, whereas the change I mentioned is for icon buttons (so no text, purely an icon on the button).
> 

Can you give me some example I could check to see the difference please? For thermostats for example I have not noticed any change for the two buttons to change mode and the other I don't remind what it does...

> Reg "Using Chrome on a desktop I still have buttons twice": You mean in Chrome they are rendered twice or the screen reader is listing/saying them twice (if you can verify that distinction at all)?

By pressing tab key I have them twice, except for "check configuration" even if they are displayed once.
Using arrow is the same, as there were two elements.

For the main reported bug, strangely today I cannot reproduce. The only thing I updated before noticing the change is HACS...
And by talking with some of my friends about other web interfaces as they don't use Home Assistant, it looks like Google broke a lot of things in chrome on Android.

**Comment by Nardol** [comment]
I still have the two elements but clicking works.
As for the type, it looks like a strangeness from Google. Sometimes TalkBack recognizes buttons or tabs, sometimes the types are not announced.
